test-icons.html

```diff
@@ -9,11 +9,9 @@
 
     <body>
         <!-- Notes - test in browser -->
-        <p>Create a symlink for the icon asset locations:</p>
-        <p><code>mkdir node_modules/oleg-brand-kit && ln -s ../../rebuild node_modules/oleg-brand-kit/rebuild</code></p>
-        <p>Compile the SASS files:</p>
-        <p><code>sass --watch 'rebuild/_config.scss':'test-icons.css' --style compressed</code></p>
-        <p>And open the test-icons.html file in your browser</p>
+        <h2>Preview Instructions</h2>
+        <p><code>sass --watch 'test-icons.scss':test-icons.css --style compressed</code></p>
```

Comment:
Am I supposed to be able to run this command? I get `zsh: command not found: sass`

Reply:
Just means you don't already have sass installed – `npm install -g sass`